scripts/gh-pages-deploy.sh

```diff
@@ -62,12 +62,3 @@ node shell/closure.js "${dest_dir}/lite-xl-files.min.js" "${dest_dir}/lite-xl-fi
 # these files can be removed for final distribution
 rm -rf "$(pwd)/${dest_dir}/lite-xl.js" "$(pwd)/${dest_dir}/lite-xl-files.js" \
         "${data_dir}" "$(pwd)/${dest_dir}/doc"
-
-package_name="github-pages"
-echo "Creating a compressed archive ${package_name}"
-
-tar \
```

Comment:
Have you tested this to work with the GH pages publishing workflow? It doesn't seem to create the tar file for publishing.

Reply:
The tar is now created by https://github.com/lite-xl/playground/blob/0a56d8c7ca427cfb4e7c64592b7e9bc72b9f91b7/.github/workflows/deploy.yml#L72-L73

I haven't tried actually publishing, but the artifact that gets created is basically the same to the one this repo creates.
Example action result: https://github.com/Guldoman/playground/actions/runs/11261084291